README.md

```diff
@@ -1 +1,5 @@
-# decimal-number
\ No newline at end of file
+# Number of ones and zeros in a given decimal number:
+- In This  program, we are reading the decimal number using ‘Ch’ variable.
+- A decimal number system is a base 10 number system using digits for 0 to 9 whereas binary number system is base 2 and uses 0 and 1. 
+- Check whether the number is less than or equal to zero. Divide the number  and store the remainder in the array. 
```

Comment:
README should say only about application and not about coding, so change accordingly